### DISCUSSION TITLE
[Projects beta] Add a new field that is “multi-select people” similar to “assignee” but is not the assignee field

### DISCUSSION THREAD

**Discussion by markgammon** [community/community#10045] [Projects and Issues]
We want to set a team of people that will work on any given issue, but we use assignees in issues for the person doing the current work needed and pass this role back and forth between team members. We'd like a way to set a team of people that draws on the database of team members, and has avatars, like assignee.

**Comment by dprada** [comment]
We have the same need. We would like to include the field "Involved" to enlist the people that are already, or were, involved in a task.
Imagine, for instance, a task was initially assigned to X and Y when the status was "To do". And Y is the only person working on it when the status changes to "In progress". We would like to keep track of initial assignments and effective assignments ("Involved").

**Comment by ccolella-mdc** [comment]
I feel like being able to add a custom field that is a single select for a user would be very good as well. We like to assign different people to do different things for an issue so it would be nice to be able to separate those. Sometimes those need to be a single person, and sometimes multiple.

**Comment by hianhianhian** [comment]
We have a similar use case as well! This would be great to have

**Comment by rrussell** [comment]
I good idea to create a new custom field type for this I guess, called Multiple Select for example 🤓
<img width="529" alt="Screen Shot 2023-03-31 at 17 34 40" src="https://user-images.githubusercontent.com/2603594/229224514-8ad8a416-cbfd-434e-a9d2-2cbeb7124379.png">

**Comment by ahmetsenturk** [comment]
That would be very helpful indeed. In most agile projects, assigning only one responsible person to an issue is expected. But still, sometimes people work together in some cases. So, it would be great to have a field like collaborators where one can select multiple team members.

**Comment by SindhuSreedharan** [comment]
Is this feature being prioritized for execution? We would like to have this option as well.

**Comment by alanosman** [comment]
Definitely a use case.  I would start by simply adding ticket members and then they can pass it around to each other.  Some tickets like this require a back end and front end development effort, with different team members.   Why do we have to write two tickets for each?  having multiple assignees would solve that issue.

**Comment by davidbodow-st** [comment]
Would be helpful to segment out different types of users' relationships to the project (e.g. not just assignee/reviewer as currently exists, but also product manager, etc)

**Comment by RisingDusk** [comment]
I also need this for my projects so that I can associate other users to a ticket.

**Comment by benjamin-popp-zap-llc** [comment]
I went to custom fields specifically to create a "manager" field. Was disappointed that I couldn't create a field that lets me put in a person.

**Comment by samuelpatro** [comment]
This will be nice feature to have i want to assign tasks for people who should do that task and other as collaborators or reviewers but i cant :(

**Comment by gotpredictions** [comment]
I am looking to assign product managers to issues and create a custom field and looks like it is not possible yet.

**Comment by lfpp** [comment]
With the recent rollout of Issue fields (now GA) and multi-select fields, this gap feels more relevant than ever. We now have Text, Number, Date, Single select and Multi-select as field types, but still no way to reference an actual GitHub user/org member in a custom field, the same way Assignees does natively.

Our use case: we split tasks into two phases handled by different people — one person analyzes/defines the task, another implements it — and we track time spent on each phase separately. We'd like a "Refined by" / "Analyzed by" field that behaves like Assignees (autocomplete, avatar, real user reference), so we can filter, group, and report by person for each phase without relying on a free-text field or a single-select list we have to maintain manually as the team changes.
